### DISCUSSION TITLE
so few of their like seem to emerge from the modern world of computing (explanation on D. Ritchie and on behavior of Stallman)

### DISCUSSION THREAD

**Discussion by AnimusPEXUS** [rms-support-letter/rms-support-letter.github.io#5909] [General]
Again, this is My speculation, and I not pretending this to be the ultimate truth . 

someone said in Discord on D. Ritchie: "so few of their like seem to emerge from the modern world of computing."

I want to express what I think about this, and how I explain RMS's behavior.

this is because people have other problems.. politics and culture nurtures consumption culture.

I again and again blaming leftism on this, especially feminism: in the core of all society problems - there are women, who can betray husband in marriage without any consequences to here - this leads to indifferent view on marriage by men - and indifferent view on life in general: men no longer sees woman as reliable ally. - this is what actually leads to corruption and many society problems.

those problems also causes problems in responsibility in software and hardware development, so people can not concentrate on development without some forcing entity, like company/corporation

so, instead concentrating on some computer problems, on free software, many have 'real life' problems, like 'where to get money for living' and 'how to satisfy my goddess' - farther more: if your goddess will receive new feministic program 'Stallman hates women' - you have to comply to stand against Stallman, or get divorced

this is how companies, like at&t and google make business: they create appropriate environment, so people can concentrate on job and not to think of any other problems. - but this also lead people to profanity in other matters of live: gcc/fedora/rh/ibm/etc workers - maybe an awesome programmers, but they appeared completely disconnected from fsf/gnu/gcc goals

this also explains why RMS behaves as he behaves. I think it's dissatisfaction in (personal) life in the flash. so who to blame in RMS's behavior is BigTech companies in the first place, who creates consumption culture, where husband have to earn lots (piles) of money to satisfy his goddess and here consumption needs, and not to concentrate on real world problems.

note: nukeop exercises freedom of speech violations: https://t.me/AnimusPEXUSChan/276 https://t.me/AnimusPEXUSChan/277

**Comment by jobbautista9** [comment]
Can we please stop blaming leftism and feminism here? It doesn't make sense.

A true feminist would defend RMS, as he consistently stood up for the rights of women.

A true leftist would defend RMS, as they see the importance of his ideas for the left. Free software has saved Venezuela's oil industry under Chavez after all.

Those "woke" people who are attacking him are not really leftists nor feminists. They only pretend to be one. The fact that they collaborate with their supposed enemy, that is the corporations, just screams the fact that they are really just liberals who want to destroy the left.

So stop falling for their stupidpol trap. By blaming the left for this, you only confirm the enemy's narrative that only right-wingers support RMS. This is a bipartisan issue.

**Reply by AnimusPEXUS** [reply]
> Can we please stop blaming leftism and feminism here?

if this is only the question, then answer is - no. if this is prohibition - then I'll think about this..

> It doesn't make sense.

I don't think so

>A true feminist

>A true leftist

I doubt this

>They only pretend to be one.

and this too

>liberals who want to destroy the left

aren't liberals also left?

>So stop falling for their stupidpol trap.

I'm not falling into trap. I know, what Stallman is leftist and I think this is one of he's fundamental mistakes.
I simply trying to analyze, what is truly lead to this situation. not only from perspective of somebody trying to blow a problem, but also from a perspective of RMS's behavior. 

Who else doesn't want this discussion to continue, please select thumbs down emoticon.
Who want to continue, and, possibly find some solution to a situation - thumbs up.
who doesn't care - you know who what to do with beer and popcorn.

**Reply by jobbautista9** [reply]
>>A true feminist

> I doubt this

I'm not sure why you're doubting. Nadine Strossen, who is a former president of the ACLU, is a feminist, and she defended Stallman in the [Cancel We the Web article](https://www.wetheweb.org/post/cancel-we-the-web).

>>A true leftist

> I doubt this

Free software has been well received by Latin American countries like Venezuela, which are mostly leftist. So I see this as a definite possibility.

> aren't liberals also left?

If we're talking about the early times where feudalism and the church was dominant, yes.

Nowadays, no. They defend the status quo that is capitalism. Leftists seek to overthrow it. So clearly liberals are not leftists. They are closer to the center of the political spectrum.

RMS is not really a leftist, he's more of a social democrat (which are not leftists despite how they are being portrayed by most. SDs seek to reform capitalism. They are a lot better than liberals though).

This is why I'm telling you that you are falling for their trap. This is clearly not a unipartisan issue.

**Reply by AnimusPEXUS** [reply]
ok.. I see we have some incompatibility in seeing things.
who do you think BigTech is - is it right wing or left wing in your opinion?
also, who, do you think, stands behind attack on Stallman? - is it leftists? liberals? rightists? government? BigTech? who?
also, I would like to hear your version on question 'why Stallman behaves like he behaves?' - note: i'm not blaming him in anything

**Reply by franpoli** [reply]
People from all walks in life is concerned by fair justice. There is no reason to split us in various political hues. Identify the interests of your opponents and you may understand the unmentionable reasons for their attacks.

**Reply by jobbautista9** [reply]
> who do you think BigTech is - is it right wing or left wing in your opinion?

They are right-wing in my opinion, as they seek to retain capitalism and the status quo.

> also, who, do you think, stands behind attack on Stallman? - is it leftists? liberals? rightists? government? BigTech? who?

Big Tech definitely wants him removed, and so does the government most likely. I don't want to turn this into an unipartisan issue. I don't care what your damn political affiliation is.

You're a conservative/right-winger and you support Stallman? Good.
You're a socialist/leftist and you support Stallman? Good.
You're a fascist/Nazi and you support Stallman? Eh, maybe you should reconsider your beliefs, but I appreciate the support.
You're a fascist/Nazi and you want Stallman removed? Fuck you.
You're a socialist/leftist and you want Stallman removed? Fuck you, and you must reconsider whether you are really a leftist or not.

**Reply by AnimusPEXUS** [reply]
>They are right-wing in my opinion, as they seek to retain capitalism and the status quo

and in my opinion, BigTech is overcrowded with leftists - this is obvious if you remember their support to BLM and rainbow people and how Intel and other corporations have become dependent on China

>I don't want to turn this into an unipartisan issue

I don't ask you to. but I wan to know exact people, who initiated and supported this anti-Stallman movement and, when their names will be listed, I think what court have to decide whatever they guilty of something or not.

now as you decided to enrage on me personally, I'll clarify my own position in this situation.

I'm not rightist - because I against faith and love and any other scientifically indeterminable things. but I support private property and I think that husband have to be the main in family - patriarchy. also I'm against nazism. and I support abortions in case of medical and scientific proof of failure of fides.

I'm not a leftist - because I'm against rainbow people; against feminists; against transgenders; against abnormal relationships in families, like open marriages, adultery, divorces  - all those I consider a mental illness, and mentally ill people, should not take part in political decisions as a minimum, and maybe there should be some more restrictions for them.

Now, why I support Stallman: he is really great philosopher and he precisely foreseen future of proprietary software. He devoted him self to develop such a huge goal - this deserves respect.

And to the criticism of Stallman: the blind truth about GNU on it's own, is that it appearing not able to provide complete operating system. GNU on it's own is unable to make money and text to the song "when we'll have enough free software.. we'll kick out those dirty licenses" - this is so altruistic - yet if you can't find money to feed developers and their needs - this song remains only and altruistic __song__. GNU had to have more sharp teeth and belonging of GCC to GNU and FSF should be more stricter.. and what Stallman deed? - he just estranged him self from everything. So, I don't think Stallman is so good at steering and making GNU more success full. 

I don't say Stallman should be removed from everywhere. But the truth is, what he isn't good enough to make GNU self-sufficient.

Stallman is really-really good in one particular thing: in detecting threats to free software - he is master at this. 

and, by the way, Stallman is not forever. and you guys? - you just afraid to call names! who are you without Stallman? just a bunch of headless puppets, who felt sorry for Stallman, but at the same time, You afraid to pinpoint the forces who started all this anti-Stallman wave. 'I don't want to turn this into an unipartisan' - and what if it will turn out it really is unipartisan? - what then? will you pass?.. My guess is: you don't want to mess with politics - this may be understandable, but more of this, this is also ignorant.

damn, now I'm filling like I'm talking to a bunch of angry kids, who don't know there to apply their force to.

**Reply by 6r1d** [reply]
Can we possibly __not__ have a big mood moment, please?

**Reply by AnimusPEXUS** [reply]
yes-yes - shaming people for having their opinion and banning them for this is - sooo leftistic, tovarish, @6r1d

**Reply by 6r1d** [reply]
> banning them for this is - sooo leftistic, tovarish, @6r1d

Ahem. Did I ban anyone in this repo, at all? Actually, I've tried to talk to you in Discord, but I can't.

**Reply by AnimusPEXUS** [reply]
> Ahem. Did I ban anyone in this repo, at all? Actually, I've tried to talk to you in Discord, but I can't.

well, I'm already used to be banned from everywhere because of having my own opinion on things. and from discord I left by my own, because I felt my thoughts are undesirable. and I will leave this repository too. 

you can talk to me in both languages, I didn't knew you were Russian when you were talked to me about developing compiler.

**Reply by 6r1d** [reply]
I mean I have three keyboard layouts, switching them is getting tedious!
Anyway, I think you both are missing the point. There was no need for more aggressive tone, no need to react to it and it's a time we (in my opinion) should use to concentrate on code, not on politics and arguments.
I am not telling this to insult either of you, because we all need to stay together to have more impact. I am not telling you what to do, you are free to do your choices and if you want to investigate, do that.

> well, I'm already used to be banned from everywhere

Not by me. I was discussing CISC processor designs, doing some electronics, etc. It is sad you were banned somewhere, but so far we've only discussed the fate of free software and hardware.

> and I will leave this repository too.

Will you stay in Matrix? Or you don't use it?

**Reply by AnimusPEXUS** [reply]
I don't use matrix, as I believe it is vulnerable to the man in the middle and possibly other attacks and blockings, just like e-mail and xmpp.

sorry, @6r1d, but I decided to continue to work alone, to avoid any possible (re)licensing issues in future (which may arise because of my character). I will develop everything in GPLv3 or more stricter licenses which Stallman may release. besides, solitude work, imo, is more productive, as I will not have to count on anybody and will avoid bureaucracy momentum as such.

**Reply by 6r1d** [reply]
> sorry, @6r1d, but I decided to continue to work alone, to avoid any possible (re)licensing issues in future (which may arise because of my character).

Well, I wish you the best, have fun doing that.
